RELEASE.md

```diff
@@ -0,0 +1,12 @@
+To do consistent releases, we use the following steps:
+
+1. Create a new issue and branch.
+1. Update the version number for the mono repo in the `package.json` file.
```

Comment:
```suggestion
1. Update the version number for the mono repo in the `lerna.json` and `package.json` file.
```

Reply:
Lerna updates itself and doesn't need a manual update.